integration/src/main/java/scripts/testscripts/PrivateControlledGcsBucketLifecycle.java

```diff
@@ -124,15 +124,14 @@ public void doUserJourney(TestUserSpecification testUser, WorkspaceApi workspace
     BlobId blobId = BlobId.of(bucketName, GCS_BLOB_NAME);
     BlobInfo blobInfo = BlobInfo.newBuilder(blobId).setContentType("text/plain").build();
 
-    // TODO(PF-633): workspace owners can write to private buckets via "roles/storage.admin".
-    // // Owner cannot write object to bucket
-    // StorageException ownerCannotWritePrivateBucket =
-    //     assertThrows(
-    //         StorageException.class,
-    //         () -> ownerStorageClient.create(blobInfo, GCS_BLOB_CONTENT.getBytes()),
-    //         "Workspace owner was able to write to private bucket");
-    // assertEquals(HttpStatusCodes.STATUS_CODE_FORBIDDEN, ownerCannotWritePrivateBucket.getCode());
-    // logger.info("Workspace owner cannot write to private resource as expected");
+    // Owner cannot write object to bucket
```

Comment:
I thought owners (and other workspace users) should be able to write to their own private bucket. And so I think this is testing whether the owner can write to a bucket that is private to another user. Is that correct? If so, I think just spelling that out a bit more in the comment would be helpful.

Reply:
Yep, you've got it exactly. Clarified the comments a bit